core/server/helpers/body_class.js

```diff
@@ -8,19 +8,19 @@ var proxy = require('./proxy'),
 
 // We use the name body_class to match the helper for consistency:
 module.exports = function body_class(options) { // eslint-disable-line camelcase
-    var classes = [],
-        context = options.data.root.context,
-        post = this.post,
-        tags = this.post && this.post.tags ? this.post.tags : this.tags || [],
-        page = this.post && this.post.page ? this.post.page : this.page || false;
+    let classes = [];
+    const context = options.data.root.context;
+    const obj = this.post || this.page;
+    const tags = obj && obj.tags ? obj.tags : [];
```

Comment:
Github lost this comment somehow :shrimp: 

**NABD**

You can just use boolean operators rather than ternary for these, e.g. 
```js
const tags = obj && obj.tags || [];
const isPage = !!(obj && obj.page);
```

Doesn't make much difference, though the `isPage` assignment might be better written to be explicit about the type of the data, e.g. above or this:
```js
const isPage = obj && obj.page ? true : false;
```

That shows we're deffo assigning a boolean, I had to do a minor check in my head that `obj.page` was gonna be a boolean :man_shrugging:

Reply:
updated